app/onboard/page.tsx

```diff
@@ -1,5 +1,121 @@
-import OnboardView from '@/src/views/onboard'
+'use client'
 
-export default function page() {
-  return <OnboardView />
-}
\ No newline at end of file
+import Image from 'next/image'
+import splashLogo from '@/src/assets/images/(logo)/splash_logo.svg'
+import { useEffect, useState } from 'react'
+import { FieldErrors, useForm, UseFormRegister } from 'react-hook-form'
+import { useGetMyProfile, useUpdateUser } from '@/src/entities/user'
+import { useRouter } from 'next/navigation'
+import { HelperText, Spacer } from '@/src/shared/ui'
+
+export default function Page() {
+  const router = useRouter()
+  const [imageLoaded, setImageLoaded] = useState(false)
+
+  const { mutate: updateUser } = useUpdateUser()
+  const { data: profile } = useGetMyProfile()
+
+  const {
+    register,
+    handleSubmit,
+    setValue,
+    formState: { errors },
+  } = useForm({
+    defaultValues: {
+      nickname: profile?.name || '',
+    },
+  })
+
+  useEffect(() => {
+    if (profile) {
+      setValue('nickname', profile.name)
+    }
+  }, [profile, setValue])
+
+  const onSubmit = async (data: { nickname: string }) => {
+    updateUser(
+      { name: data.nickname, profile_url: '', description: '' },
+      {
+        onSuccess: () => {
+          router.push(`/`)
+        },
+      }
+    )
+  }
+
+  return (
+    <div
+      className={`w-full h-screen flex flex-col items-center justify-center
+      transition-all duration-[2000ms] ease-in-out ${
+        imageLoaded
+          ? 'bg-brand'
+          : 'bg-gradient-to-t from-brand to-container-light-blue'
+      } `}
+    >
+      <Image
+        className={`w-[450px] h-auto m-[-200px] fixed bottom-0 transition-all duration-[2000ms] ease-in-out ${
+          imageLoaded
+            ? 'scale-[150%] -translate-y-[50vh] opacity-10'
+            : 'scale-[100%]  opacity-100'
+        }  `}
+        src={splashLogo}
+        alt='splash'
+        width={100}
+        height={100}
+        onLoadingComplete={() => setImageLoaded(true)}
+      />
+
+      <form
+        className='w-full h-full flex flex-col items-center justify-center z-10'
+        onSubmit={handleSubmit(onSubmit)}
+      >
+        <NicknameInput register={register} errors={errors} />
+        <button
+          type='submit'
+          className='text-white text-[15px] w-fit font-extrabold bottom-[144px] items-center justify-center absolute left-1/2 transform -translate-x-1/2'
+        >
+          시작하기
+        </button>
+      </form>
+    </div>
+  )
+}
+
+interface NicknameInputProps {
+  register: UseFormRegister<{ nickname: string }>
+  errors: FieldErrors<{ nickname: string }>
+}
+
+function NicknameInput({ register, errors }: NicknameInputProps) {
+  const validateNickname = (value: string) => {
+    if (!value) return '닉네임을 입력해주세요.'
+    if (value.includes(' ')) return '공백은 입력할 수 없습니다.'
+    if (/[`~!@#$%^&*|\\'"/?]/gi.test(value))
```

Comment:
코드 읽는 중에 의문이 생겼는데, 
특수 문자 제한 범위가 적절한지 이야기 나눠보고 싶습니다!
- 기존: 일부 특수문자를 제한 
- 제안: 한글/영어/숫자 화이트리스트 처리 
```suggestion
    if (/[^a-zA-Z0-9가-힣]/.test(value))
```